@kiva/kv-components/vue/KvLocationSelector.vue

```diff
@@ -213,6 +214,11 @@ export default {
 			return total;
 		};
 
+		watch(activeIsoCodes, (val) => {
+			selected.value = val;
```

Comment:
We should probably check that the arrays of values don't have the exact same values before applying.

Reply:
Cool! Even, I'll add a watcher for the regions prop that could be changed by other filters.